queries/phpdoc/highlights.scm

```diff
@@ -0,0 +1,29 @@
+(tag_name) @tag
+(tag
+  (tag_name) @_tag (#eq? @_tag "@param")
+  (variable_name) @parameter
+)
+(tag
+  (tag_name) @_tag (#eq? @_tag "@property")
+  (variable_name) @property
+)
+(tag
```

Comment:
Please have a look into `CONTRIBTING.md` to see which nodes are available

Reply:
But `tag` is a node provided by the phpdoc parser.

https://github.com/claytonrcarter/tree-sitter-phpdoc/blob/master/grammar.js#L40

https://github.com/claytonrcarter/tree-sitter-phpdoc/blob/master/corpus/tests.txt#L30

Or what exactly is wrong here?

Reply:
Ah, sorry I was confusing the php captures in quotes with the ones from this project :face_palm: